app/DoctrineMigrations/Version20160803183748.php

```diff
@@ -8,19 +8,20 @@
 /**
  * Auto-generated Migration: Please modify to your needs!
  */
-class Version20160729173849 extends AbstractMigration
+class Version20160803183748 extends AbstractMigration
```

Comment:
I can do that in the future. I think its only necessary once an app is in production, but to learn we can do this for development too.